ksql-engine/src/test/java/io/confluent/ksql/util/KafkaTopicClientTest.java

```diff
@@ -124,6 +129,58 @@ public void shouldNotFailIfTopicAlreadyExistsWhenCreating() throws InterruptedEx
     KafkaTopicClient kafkaTopicClient = new KafkaTopicClientImpl(adminClient);
     kafkaTopicClient.createTopic(topicName1, 1, (short)1, false);
     verify(adminClient);
+
+  }
+
+
+  // Compares two collections of 'kafka.clients.admin.NewTopic' objects to verify that they are
+  // the same. The NewTopic.equals method isn't implemented so it just does reference equality,
+  // which is not what we want in these tests.
+  class NewTopicComparator implements Comparator<Collection<NewTopic>> {
+
+    @Override
+    public int compare(Collection<NewTopic> topicCollection1,
+                       Collection<NewTopic> topicCollection2) {
+      if (topicCollection1.size() != topicCollection2.size()) {
+        return -1;
+      }
+
+      Iterator<NewTopic> firstIterator = topicCollection1.iterator();
+      Iterator<NewTopic> secondIterator = topicCollection2.iterator();
+
+      while (firstIterator.hasNext()) {
```

Comment:
The order of topics may not be the same in the two collections.

Reply:
Yes. But for now, we only are testing with lists of single size, since our API creates one topic at a time. So it isn't an issue IMO.